pages/having-a-child.js

```diff
@@ -23,14 +24,101 @@ export default function lifejourney() {
       <div id="wb-bc" className="container pt-4">
         <Breadcrumb />
       </div>
-      <Accordion>
-        <div className="pr-6">
-          <MoreInfo text={t.moreInfoCommunity} />
-          <MoreInfo text={t.moreInfoFertility} />
-          <MoreInfo text={t.moreInfoHealth} />
-          <MoreInfo text={t.moreInfoFinancial} />
+
+      <div className="container flex flex-col md:flex-row align-items-center ">
```

Comment:
```suggestion
      <div className="container flex flex-col md:flex-row align-items-center items-start ">
```
The div on line 29 is taking up the entire height of the parent div on line 28, if we add ```items-start``` to the end here makes it so each of the children are only the height of their content. This makes the sticky function work as intended as larger resolutions.